src/tests/unit/handler.test.js

```diff
@@ -1,14 +1,51 @@
 'use strict'
 
-const { handler } = require('../../handler')
+const handler = require('../../handler')
 const chai = require('chai')
 const expect = chai.expect
 const httpStatus = require('http-status-codes')
+const fs = require('fs')
+const nock = require('nock')
```

Comment:
maybe better to name it httpServerMock? It was hard to google 'javascript nock', kept getting "do you main javascirpt mock"?

Reply:
Renamed